perf_test.js

```diff
@@ -0,0 +1,69 @@
+function leftpad_orginal (str, len, ch) {
```

Comment:
Sorry, I've been busy with other stuff. If you have time could you put this function in a standalone js file and name it `O(n)`? Similar to the functions below. and put them in a `bench` folder. If not I'll do it myself. Thanks.